cmd/notification_platforms.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"github.com/Scalingo/cli/cmd/autocomplete"
+	"github.com/Scalingo/cli/notification_platforms"
+	cli "github.com/Scalingo/codegangsta-cli"
+)
+
+var (
+	NotificationPlatformListCommand = cli.Command{
+		Name:        "platforms-list",
```

Comment:
I made the choice to transform it to `notification-platforms` as stated by the variable name ;)